force-app/main/default/classes/TaskService.cls

```diff
@@ -4,4 +4,46 @@ public with sharing class TaskService
     {
 
     }
+
+    /**************************************************************
+    *   Creates a task associated with the opportunity 
+    *   to follow up in 10 days time 
+    *   Called in the context of an after insert trigger
+    ***************************************************************/
+
+    public static void addFollowUpTask(Map<Id,Opportunity> oppMap)
+    {
+        //TODO: if(!Triggers.Stop) 
```

Comment:
If you are looking at adding trigger handler type of functionality, that should live in the trigger rather than the service class, otherwise your service has to know too much about where it is called from and becomes tightly coupled with triggers.